charts/solr/templates/storageclass.yaml

```diff
@@ -0,0 +1,5 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: efs-sc
```

Comment:
Do not add a StorageClass, this should be done by the CSI driver or by the cluster-admin on the cluster directly